src/lat/lattice-functions.h

```diff
@@ -74,6 +74,26 @@ bool ComputeCompactLatticeAlphas(const CompactLattice &lat,
 bool ComputeCompactLatticeBetas(const CompactLattice &lat,
                                 vector<double> *beta);
 
+
+static inline double LogAddOrMax(bool viterbi, double a, double b) {
```

Comment:
also if you are going to leave it in the header you should take away the 'static'.